File: fixtures/go_output/post_with_data_raw.go
package main

import (
	"fmt"
	"io/ioutil"
	"log"
	"net/http"
	"strings"
)

func main() {
	client := &http.Client{}
	var data = strings.NewReader(`msg1=wow&msg2=such&msg3=@rawmsg`)
	req, err := http.NewRequest("POST", "http://example.com/post", data)
	if err != nil {
		log.Fatal(err)
	}
	resp, err := client.Do(req)
	if err != nil {
		log.Fatal(err)
	}
	bodyText, err := ioutil.ReadAll(resp.Body)
	if err != nil {
		log.Fatal(err)
	}
	fmt.Printf("%s\n", bodyText)
}

File: test/fixtures/node-got/post_with_data_raw.js
import got from 'got';

const response = await got.post('http://localhost:28139/post', {
  headers: {
    'Content-Type': 'application/x-www-form-urlencoded'
  },
  body: 'msg1=wow&msg2=such&msg3=@rawmsg'
});
